knowledge-base/kb-changehandicon-to-loadingicon.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Change Hand Icon to Any Loading Icon Upon Clicking on Filter
+description: "How to change hand icon to any loading icon upon clicking on filter"
+type: how-to
+page_title: How to change hand icon to any loading icon upon clicking on filter for WPF 
+slug: kb-changehandicon-to-loadingicon
+position: 0
+tags: loading icon, hand icon, filter, busyindicator
+ticketid: 1603581
+res_type: kb
+---
+
+## Environment
+<table>
+    <tbody>
+        <tr>
+            <td>Product Version</td>
+            <td>2023.2.718</td>
+        </tr>
+        <tr>
+            <td>Product</td>
+            <td>RadBusyIndicator for WPF</td>
+        </tr>
+    </tbody>
+</table>
+
+
+## Description
+
+How to change hand icon to any loading icon upon clicking on filter.
+
+## Solution
+
+You can use the `MouseDown` event of the `RadGridView` control to get the `FilterDropDown`. Then you can override the cursor.
+
+#### __[C#]__
+{{region kb-changehandicon-to-loadingicon}}
```

Comment:
Does not contain the full approach that Marti has provided in the ticket. More specifically, the invocation of the AddHandler method on the RadGridView instance.